src/scripts/types/mhct.ts

```diff
@@ -0,0 +1,36 @@
+export interface IntakeMessage {
```

Comment:
Does this need a hunt_details component? We normally submit a message.hunt_details element (when there are details for the area/event)

Reply:
It will. It's not in yet because it's only done in the untyped main.js code. See `createIntakeMessage`.

Reply:
ok, so hunt details can still submit with the message then? I saw it was still making hunt details but was worried typescript might get angry at it or something.

Reply:
Correct. If you did need to access hunt_details from typescript, then it will need to be added.
This might be done in my details PR that's in draft.